.env

```diff
@@ -1,7 +1,11 @@
 REDIS_USER=redis
 REDIS_PASSWD=secret
+API_REDIS_USER=redis
+API_REDIS_PASSWORD=secret
 REDIS_SSL_CERTFILE=./docker/config/certs/client-cert.pem
+API_REDIS_SSL_CERTFILE=./docker/config/certs/client-cert.pem
 REDIS_SSL_KEYFILE=./docker/config/certs/client-key.pem
+API_REDIS_SSL_KEYFILE=./docker/config/certs/client-key.pem
```

Comment:
Btw you don't need a cert file, nor a password and username to make this work (actually I am not so sure about password and username)